From 35e77153831ee1ea6d3f26961f9527070edbfe8a Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.2-35e77153831-20210413T122645Z-avarab@gmail.com>
In-Reply-To: <cover-0.2-00000000000-20210413T122645Z-avarab@gmail.com>
References: <patch-1.1-f0542cd902c-20210413T121930Z-avarab@gmail.com>
	<cover-0.2-00000000000-20210413T122645Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 13 Apr 2021 13:58:32 +0200
Subject: [PATCH 1/2] diff tests: remove redundant case statement
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove a case statement whose two arms executed the same code. This
was added for i18n testing in 6dd88832e77 (diffstat summary line
varies by locale: miscellany, 2012-03-13), and was changed from
test_i18ncmp in my 1108cea7f8e (tests: remove most uses of
test_i18ncmp, 2021-02-11).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/t4013-diff-various.sh | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/t/t4013-diff-various.sh b/t/t4013-diff-various.sh
index 6cca8b84a6b..67f6411aff9 100755
--- a/t/t4013-diff-various.sh
+++ b/t/t4013-diff-various.sh
@@ -220,13 +220,7 @@ do
 		then
 			process_diffs "$actual" >actual &&
 			process_diffs "$expect" >expect &&
-			case $cmd in
-			*format-patch* | *-stat*)
-				test_cmp expect actual;;
-			*)
-				test_cmp expect actual;;
-			esac &&
-			rm -f "$actual" actual expect
+			test_cmp expect actual
 		else
 			# this is to help developing new tests.
 			cp "$actual" "$expect"
-- 
2.31.1.646.g06d606cb9c7

